N/Nemo/Compat.toml

```diff
@@ -65,7 +65,7 @@ Arb_jll = "2.17.0"
 LoadFlint = "0.2.1-0.2"
 
 ["0.18-0"]
-Antic_jll = "0.2.2-0.2"
+Antic_jll = "0.2.2-0.2.2"
```

Comment:
Why do you need to change this if the broken version is yanked?

Reply:
julia 1.0

Reply:
Julia v1.0 wouldn't use the JLL anyway

Reply:
I don't like that it will pull in 0.2.3 on julia 1.0 and 0.2.2 on julia >= 1.1. I don't like the inconsistent state.

Reply:
That seems like a strange argument, basically no package install the same version on all Julia versions :shrug:

Reply:
Which package with no restriction on the julia version in the `[compat]` section installs different versions on different julia versions?

Reply:
Which package with no restriction on the julia version in the `[compat]` section installs different versions on different julia versions?